packages/odyssey-react/src/components/Banner/Banner.tsx

```diff
@@ -114,16 +113,8 @@ export const Banner = withTheme(
             <Heading visualLevel="6" noEndMargin children={heading} />
           </div>
         )}
-        {content && (
-          <span className={styles.content}>
-            <Text as="p" children={content} />
```

Comment:
this is fine but also a long term consistency risk in that we have `<p>` and `<section>` text styles that don't match 100% with what we declare in `<Text />`. The modularity is what we want and if we find we need different styles in special cases we need to build the right abstractions around or into Text to support that. What did you need that was different here?

Reply:
The only styles being pulled in from `<Text>` here were the same styles we apply via `<Box>` and `<Link>`, which is a duplication we might want to address in the future, to avoid the extra weight.

If we wanted to use `<Text>` here, we'd need to add a `maxWidth` prop that could take custom values, which seems dangerous. Alternately, we could allow for `<Text>` to take `className`, but then it's getting further into `<Box>` territory / doesn't ensure consistency.